constants/materialStyles.ts

```diff
@@ -39,6 +40,28 @@ export const useStyles = makeStyles((theme: Theme) =>
         borderBottom: '.25rem solid green !important',
       },
     },
+    formControl: {
```

Comment:
there are a lot of absolute values here. can they be written to be more adaptive?